joomla/joomla-cms
# [4.0] Code cleanup: remove unnecessary onContentAfterSave in PlgFieldsRepeatable

### Summary of Changes

Prior to this pull request `PlgFieldsRepeatable::onContentAfterSave` was responsible for converting the field values to JSON. However `PlgSystemFields::onContentAfterSave` (see below) does the same task. This pull request removes the unnecessary processing.

https://github.com/joomla/joomla-cms/blob/27371eecdba5247fdc5344c85f616ea1b1aa8e08/plugins/system/fields/fields.php#L66-L148

### Testing Instructions

Create a custom field of type repeatable and save data to it. After the saving process the entered data should still be there.

### Expected result

Same as before this commit
